core/src/main/kotlin/com/justai/jaicf/activator/BaseActivator.kt

```diff
@@ -94,6 +95,12 @@ abstract class BaseActivator(private val model: ScenarioModel) : Activator {
         }
 
         val transitionsMap = model.transitions.groupBy { it.fromState }
-        return availableStates.flatMap { transitionsMap[it] ?: emptyList() }
+
+        val lastState = StatePath.parse(availableStates.last())
```

Comment:
Isn't it true that this lastState is always equal to currentState?

Reply:
No, we process states in reverse order from current state:
Suppose currentState = /a/b/c, then the order is following: /a/b/c -> /a/b -> /a -> /
So, lastState - is either first modal ancestor, or root